vgen/src/main/resources/io/gapi/vgen/java/java_gapic.yaml

```diff
@@ -1,8 +1,10 @@
 type: io.gapi.vgen.ConfigProto
 language: java
-language_provider: io.gapi.vgen.java.JavaGapicLanguageProvider
-snippet_files:
-- main.snip
-- settings.snip
-fragment_files:
-- fragment.snip
+templates:
+- language_provider: io.gapi.vgen.java.JavaGapicCodeLanguageProvider
+  snippet_files:
+  - main.snip
+  - settings.snip
+- language_provider: io.gapi.vgen.java.JavaGapicFragmentLanguageProvider
+  snippet_files:
+  - fragment.snip
```

Comment:
This new structure implies to me that when I run the GAPIC codegen for Java, it will generate fragments too, which would be a change in behavior and not what we want. Have you tried running the artman pipeline for Java GAPIC to make sure it doesn't generate extra stuff?